Feature: add option to disable logging when image is not found

ahmadalfy opened the pull request:
Related to https://github.com/alexlafroscia/postcss-retina-bg-img/issues/3

Making logging optional.

ahmadalfy committed:
feat: add option to disable logging when image is not found

alexlafroscia commented:
This is great — thanks for the PR!

Could you add a test case to the `postcss-plugin` file that ensures that the warning is not logged when the default configuration option is overwritten? 😁

alexlafroscia commented:
The `README` should be updated to mention the new configuration option, too!

ahmadalfy commented:
Sure, let me do that

ahmadalfy committed:
doc: add the option to readme.md

alexlafroscia commented:
Released in version `5.0.0`

ahmadalfy commented:
@alexlafroscia you were right I had to add the test. I found a problem due to the bad comparison I was doing. The option was always set to `true` and I found that from the test. I have added the test and fixed the bug. Can you revert that version and redo it again if it's not too late? Or maybe release 5.0.1?

alexlafroscia commented:
It think the best approach here would be to make a PR with those additional changes and I'll do a `5.0.1` release with the fixes.

ahmadalfy commented:
Done

ahmadalfy commented:
My apologies I was pushing the commits as I go I didn't know you'd merge it instantly after updating the readme

alexlafroscia commented:
Not your fault -- I forgot that I was waiting for tests, too!

ahmadalfy commented:
Here it the updated PR https://github.com/alexlafroscia/postcss-retina-bg-img/pull/16
